benchmark/inclusion/run.sh

```diff
@@ -10,11 +10,11 @@ cp global.prm.base temp.prm
 echo "subsection Mesh refinement" >>temp.prm
 echo "set Initial global refinement = $r" >> temp.prm
 echo "end" >> temp.prm
-../../aspect temp.prm | grep Error
```

Comment:
It seems more reasonable to assume that the user has aspect in their environment PATH than in this particular spot.